apps/dashboard/src/containers/StudentScoreReport/components/ScoreList/ScoreCard/LongitudinalChart/LongitudinalChart.vue

```diff
@@ -3,7 +3,7 @@
     <div class="h-64">
       <canvas ref="canvasRef" class="w-full h-full"></canvas>
     </div>
-    <div class="flex gap-3 justify-end mt-2 text-xs">
+    <div v-if="showSupportLevels" class="flex gap-3 justify-end mt-2 text-xs">
```

Comment:
can we change this whole div block for something like this? 
```<div v-if="showSupportLevels" class="flex justify-content-center align-items-center gap-2 mt-4 text-xs">
      <div class="ml-2 flex gap-2 align-items-center">
        <div
          class="rounded-full flex-shrink-0"
          style="width: 1rem; height: 1rem;"
          :style="{ backgroundColor: SCORE_SUPPORT_LEVEL_COLORS.ABOVE }"
        ></div>
        <span>Achieved Skill</span>
      </div>
      <div class="flex gap-2 align-items-center">
        <div
          class="rounded-full flex-shrink-0"
          style="width: 1rem; height: 1rem;"
          :style="{ backgroundColor: SCORE_SUPPORT_LEVEL_COLORS.SOME }"
        ></div>
        <span>Developing Skill</span>
      </div>
      <div class="flex gap-2 align-items-center">
        <div
          class="rounded-full flex-shrink-0"
          style="width: 1rem; height: 1rem;"
          :style="{ backgroundColor: SCORE_SUPPORT_LEVEL_COLORS.BELOW }"
        ></div>
        <span>Needs Extra Support</span>
      </div>
    </div>
```

This way the legend will look like this? 
<img width="322" height="57" alt="image" src="https://github.com/user-attachments/assets/f3409931-d8c2-4f0b-a444-fd0521c462a0" />

What do you think @lucasxsong ?

Reply:
Additionally to this, can we add something like this? I am not sure if the text is the correct, but at least have a legend for the blue?
```suggestion
    <div v-else class="flex justify-content-center align-items-center gap-2 mt-4 text-xs">
      <div
          class="rounded-full flex-shrink-0"
          style="width: 1rem; height: 1rem;"
          :style="{ backgroundColor: SCORE_SUPPORT_LEVEL_COLORS.ASSESSED }"
        ></div>
      <span>Support levels are not available for this task.</span>
    </div>
```